bokehjs/src/lib/models/dom/value_of.ts

```diff
@@ -9,10 +9,27 @@ export class ValueOfView extends DOMElementView {
   override connect_signals(): void {
     super.connect_signals()
 
-    const {obj, attr} = this.model
-    if (attr in obj.properties) {
-      this.on_change(obj.properties[attr], () => this.render())
+    const fn = () => this.render()
```

Comment:
In this case it would work. However, in general you have to pay close attention to `this` binding. Additionally `fn` may be used in multiple context which provide different arguments, which we may not care about, but we definitely don't want to pass them to `render()` or other functions or methods. Thus it's better to add a an arrow function wrapper as an idiom and forget about all of these issues. If you're concerned about performance, then this isn't performance critical section and regardless modern JS runtimes are very good at inlining such code.